app/src/main/java/org/wikipedia/captcha/HCaptchaHelper.kt

```diff
@@ -0,0 +1,101 @@
+package org.wikipedia.captcha
+
+import androidx.core.net.toUri
+import androidx.fragment.app.FragmentActivity
+import com.hcaptcha.sdk.HCaptcha
+import com.hcaptcha.sdk.HCaptchaConfig
+import com.hcaptcha.sdk.HCaptchaDialogFragment
+import com.hcaptcha.sdk.HCaptchaTheme
+import com.hcaptcha.sdk.HCaptchaTokenResponse
+import org.wikipedia.WikipediaApp
+import org.wikipedia.settings.RemoteConfig
+import org.wikipedia.util.log.L
+import kotlin.String
+
+class HCaptchaHelper(
+    private val activity: FragmentActivity,
+    private val callback: Callback
+) {
+    interface Callback {
+        fun onSuccess(token: String)
+        fun onError(e: Exception)
+    }
+
+    private var hCaptcha: HCaptcha? = null
+    private var tokenResponse: HCaptchaTokenResponse? = null
+
+    private val configDefault get() = RemoteConfig.RemoteConfigHCaptcha(
+        baseURL = "https://meta.wikimedia.org",
+        jsSrc = "https://assets-hcaptcha.wikimedia.org/1/api.js",
+        endpoint = "https://hcaptcha.wikimedia.org",
+        assetHost = "https://assets-hcaptcha.wikimedia.org",
+        imgHost = "https://imgs-hcaptcha.wikimedia.org",
+        reportApi = "https://report-hcaptcha.wikimedia.org",
+        sentry = false,
+        siteKey = "e11698d6-51ca-4980-875c-72309c6678cc"
```

Comment:
Looks like the `siteKey` is outdated?
<img src="https://github.com/user-attachments/assets/4be309ed-596a-45e5-ab01-12d1e5ae6e30" width="200" />